### ISSUE TITLE
Unable to run local iOS environment

### ISSUE THREAD

**Issue by renahlee** [bluesky-social/social-app#634]
**Describe the bug**

@hs4man21 is unable to run the iOS environment.

**To Reproduce**

1. Follow installation instructions from README:

    ```
    yarn global add detox-cli
    brew tap wix/brew
    brew install applesimutils
    yarn add expo
    git clone git@github.com:bluesky-social/atproto.git
    cd atproto
    yarn
    cd packages/dev-env && yarn start
    ```

    In a separate terminal:

    ```
    rbenv install 2.7.6
    sudo gem install cocoapods
    yarn ios
    ```

**Expected behavior**

I expected to be able to run the local environment.

**Screenshots**

```
› Executing Bluesky » Upload Debug Symbols to Sentry

:x:  error: API request failed


    Run script build phase '[CP-User] Generate app.manifest for expo-updates' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'EXUpdates' from project 'Pods')

› 1 error(s), and 5 warning(s)

CommandError: Failed to build iOS project. "xcodebuild" exited with error code 65.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Details**

 - Platform: iOS
 - Platform version:
 - App version: 1.28

**Additional context**

He also:

1. Set up Sentry account and ran `SENTRY_AUTH_TOKEN="TOKEN_VALUE_HERE" yarn expo run:ios --device` from `social-app` root.
1. Attempted to update `app.json` but received the following:

    ```
    $ eas update     
    Entity not authorized: AccountEntity[<REDACTED>] (viewer = RegularUserViewerContext[<REDACTED>], action = READ, ruleIndex = -1)
    Request ID: <REDACTED>
    ```

**Comment by byCedric** [comment]
You probably have to update the Sentry organization settings if you want to upload the sourcemap to your own Sentry instance. It's defined over here: https://github.com/bluesky-social/social-app/blob/main/app.json#L91-L99

Or remove the `sentry-expo` plugin from the `app.json` file, if you don't want to use Sentry.

> Disclaimer: I'm not BlueSky related, I work with Expo 😄

**Comment by llllvvuu** [comment]
> Or remove the `sentry-expo` plugin from the `app.json` file, if you don't want to use Sentry.

This + removing the postpublish hook is sufficient IME. But I had to `rm -rf ios` and rebuild for it to take effect. #661

**Comment by ansh** [comment]
This was fixed with #749
